### DISCUSSION TITLE
Automatic Filament Retraction

### DISCUSSION THREAD

**Discussion by eboston** [X1Plus/X1Plus#394] [General]
The standard X1P firmware has an annoying habit of retracting the filament during a single color print.  Bambu Lab gave a reason that makes no sense at all as to why they do this.

What I would like to know is if the X1Plus firmware gives an option to disable this annoying behaviour?  This causes serious problems when the spool is close to empty and causes motor overload errors.

**Comment by magyoshaa** [comment]
i think you answered you own question. if it detects that the filament is binding it unloads/retracts to avoid motor damage and to try and save your print. disabling that action would likely just end up with it air printing and chewing the filament up. so it retracts it and then trys to reload to get a smooth run.  as spools neat the end the curvature of the filament becomes tighter and if your ptfe tube has to much resitance then end result is the symptoms you have.
your easiset solutions are eithr check your pipes, or if it persists run problematic spools directly instead of via the ams as then there is no retraction that would take place
